config/nuxt/modules/nuxt-pwa.js

```diff
@@ -0,0 +1,28 @@
+const { appConfig, locales } = require('../lib/data')
+const defaultLocale = locales[0]
+
+/**
+ * @see https://pwa.nuxtjs.org/
+ */
+module.exports = [
+  '@nuxtjs/pwa',
+  { // https://pwa.nuxtjs.org
+    manifest: {
+      name: appConfig.title,
+      short_name: appConfig.title,
```

Comment:
our title is currently too long to fit on my homescreen. but this is not a blocker. I think people using the kit should just hard-code it here. for use, this is fine.